evap/static/ts/src/notebook.ts

```diff
@@ -0,0 +1,72 @@
+declare const bootstrap: typeof import("bootstrap");
+import { assertDefinedUnwrap } from "./utils.js";
+import { CSRF_HEADERS } from "./csrf-utils.js";
+
+const NOTEBOOK_COOKIE_NAME = "evap_notebook_open";
+
+if (localStorage.getItem(NOTEBOOK_COOKIE_NAME) == "true") {
+    new bootstrap.Collapse(assertDefinedUnwrap(document.getElementById("notebook")));
+    new bootstrap.Collapse(assertDefinedUnwrap(document.getElementById("notebookButton")));
+    onShowNotebook();
+}
+
+assertDefinedUnwrap(document.getElementById("notebook-save-button")).addEventListener(
+    "click",
+    function (event: MouseEvent): void {
+        event.preventDefault();
+        const form = assertDefinedUnwrap(document.getElementById("notebook-form")) as HTMLFormElement;
+        const data = new FormData(form);
+
+        const target = assertDefinedUnwrap(event.target) as HTMLButtonElement;
+        const default_label = assertDefinedUnwrap(target.getAttribute("value"));
+        const sending_label = assertDefinedUnwrap(target.getAttribute("data-label-sending"));
+        const error_label = assertDefinedUnwrap(target.getAttribute("data-label-error"));
+        const cooldown_label = assertDefinedUnwrap(target.getAttribute("data-label-cooldown"));
+        target.disabled = true;
+        target.setAttribute("value", sending_label);
+        const cooldown_time = 2000;
```

Comment:
I don't really think the variable here adds much value:
* It's not clear what cooldown it is about
* It's only used once.
* When reading the setTimeout call, users have to first find the definition of the (constant) variable to interpret what is going on.

-> I'd just inline the 2000 into the `setTimeout` call. What do you think?